netaddr.go

```diff
@@ -1034,10 +1034,14 @@ func MustParseIPPrefix(s string) IPPrefix {
 // Masked returns p in its canonical form, with bits of p.IP not in p.Bits masked off.
 // If p is zero or otherwise invalid, Masked returns the zero value.
 func (p IPPrefix) Masked() IPPrefix {
-	if m, err := p.IP.Prefix(p.Bits); err == nil {
-		return m
+	if !p.Valid() {
```

Comment:
why is this necessary? won't IP.Prefix return an error if it's invalid?

Reply:
Doh, so it does. Reverted.